src/kernels/kernel.node.ts

```diff
@@ -67,6 +67,13 @@ export class Kernel extends BaseKernel {
     }
 
     protected async getDebugCellHook(): Promise<string[]> {
+        const useNewDebugger = this.configService.getSettings(undefined).useJupyterDebugger === true;
+        if (useNewDebugger) {
```

Comment:
For testing purposes, so that we can test debugging (even local kernels) with jupyter protocol.
This is much easier than spinning up a jupyter server and connecting to it for testing the new debugger.